src/components/Card.js

```diff
@@ -29,6 +35,7 @@ CardTitle.propTypes = {
   component: PropTypes.string,
   blockClassName: PropTypes.string,
   headingType: PropTypes.oneOf(['title', 'subtitle']),
+  modifiers: PropTypes.array,
```

Comment:
Try to make prop types as explicit as possible - these modifiers are a known, finite list of possible values:

```suggestion
  modifiers: PropTypes.arrayOf(
    PropTypes.oneOf(['padded'])
  ),
```